lightly/openapi_generated/swagger_client/models/jobs_data.py

```diff
@@ -0,0 +1,258 @@
+# coding: utf-8
+
+"""
+    Lightly API
+
+    Lightly.ai enables you to do self-supervised learning in an easy and intuitive way. The lightly.ai OpenAPI spec defines how one can interact with our REST API to unleash the full potential of lightly.ai  # noqa: E501
+
+    OpenAPI spec version: 1.0.0
+    Contact: support@lightly.ai
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+import pprint
```

Comment:
is this a new dependency?

Reply:
print is part of the standard library, see https://docs.python.org/3/library/pprint.html